lib/capybara-screenshot/saver.rb

```diff
@@ -26,10 +26,13 @@ def initialize(capybara, page, html_save=true, filename_prefix='screenshot')
       end
 
       def save
-        # the current_path may raise error with selenium
-        if !current_path.empty?
-          save_html if @html_save
-          save_screenshot
+        current_path do |path|
+          if !path.empty?
+            save_html if @html_save
+            save_screenshot
+          else
+            warn 'WARN: Screenshot could not be saved. `page.current_path` is empty'
```

Comment:
If `current_path` raise an exception, wouldn't we now have two warnings sent to STDOUT?